libvirt/tests/src/virtual_network/qemu/transfer_file_over_ipv6.py

```
@@ -0,0 +1,199 @@
+        )
+        if link_local_ipv6_addr is False and addresses[vm] is None:
+            test.cancel("Your guest can not get remote IPv6 address.")
+
+    # prepare test data
+    guest_path = "{0}src-{1}".format(tmp_dir, utils_misc.generate_random_string(8))
+    dest_path = "{0}dst-{1}".format(tmp_dir, utils_misc.generate_random_string(8))
+    host_path = os.path.join(
+        test.tmpdir, "tmp-%s" % utils_misc.generate_random_string(8)
+    )
+    test.log.info("Test setup: Creating %dMB file on host", filesize)
+    process.run(dd_cmd % (host_path, filesize), shell=True)
```

Ruff (0.14.8)

Function call with `shell=True` parameter identified, security issue

(S604)